lib/ansible/module_utils/network/ios/utils/utils.py

```diff
@@ -148,6 +203,31 @@ def validate_n_expand_ipv4(module, want):
     return ip_addr_want
 
 
+def netmask_to_cidr(netmask):
```

Comment:
I think we can reuse ipaddress module here.

Reply:
ok.. I'll check ipaddress implementation and update it here

Reply:
Not required for acls module, have removed the fn.